### PR TITLE
Caching timeout

### PR DESCRIPTION
**Pull request by paulauus**
Added a 15-minute timeout to the cache, so that might fix the database not updating, hopefully.

### CODE REVIEW

**Review by Leon-JS02 — APPROVED** [review]
Interesting change, hopefully this will solve the issue of the data not updating.

**Review by Leon-JS02 — APPROVED** [review]
Reapproval - removed unneeded load_dotenv().

File: dashboard/sl_queries.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def get_connection() -> connect:
                    database=ENV["DB_NAME"])
 
 
-@st.cache_data
+@st.cache_data(ttl=900)
```
Comment (mehrdadhalali) [inline dashboard/sl_queries.py:23]: The following functions could all be in the same place, so you won't have to duplicate this.

**Review by mehrdadhalali — APPROVED** [review]

File: dashboard/sl_queries.py (hunk 1 above)
Comment (mehrdadhalali) [reply dashboard/sl_queries.py:23]: Nevermind, I just realised this is a decorator.

**Pull request merged**